tools/postgres/schema.sql

```diff
@@ -349,6 +349,26 @@ CREATE TABLE webknossos.organization_usedStorage(
   PRIMARY KEY(_organization, _dataStore, _dataset, layerName, magOrDirectoryName)
 );
 
+-- Create the enum type for transaction states
+CREATE TYPE webknossos.credit_transaction_state AS ENUM ('Pending', 'Completed', 'Refunded', 'Revoked', 'PartiallyRevoked', 'Spent');
+
+-- Create the transactions table
+CREATE TABLE webknossos.organization_credit_transactions (
+    _id CHAR(24) PRIMARY KEY,
+    _organization VARCHAR(256) NOT NULL,
+    credit_change DECIMAL(14, 4) NOT NULL,
+    refundable_credit_change DECIMAL(14, 4) NOT NULL CHECK (refundable_credit_change >= 0), -- Ensure non-negative values
```

Comment:
How did you decide that this precision 4 is what we need?

Reply:
In the linked discussion I always read millicredits as the integer unit. Wouldn’t that be Decimal(14,3)? Is the current plan to render four decimal places to the user? Feels like a bit much to me, tbh. (But maybe that’s just me, as I’m used to dollars and euros, which tend to be rounded to centieuros, resulting in just two decimal places)

Reply:
Yeah, I corrected it now to `Decimal(14,3)`. Same for the `CreditTransaction.scala`